lib/msal-common/src/utils/Constants.ts

```diff
@@ -34,14 +36,20 @@ export const Constants = {
     S256_CODE_CHALLENGE_METHOD: "S256",
     URL_FORM_CONTENT_TYPE: "application/x-www-form-urlencoded;charset=utf-8",
     AUTHORIZATION_PENDING: "authorization_pending",
-    NOT_DEFINED: "not_defined"
+    NOT_DEFINED: "not_defined",
+    // Default time to throttle RequestThumbprint in milliseconds
+    DEFAULT_THROTTLE_TIME_SECONDS: 60,
```

Comment:
Should we add a separate Throttling object in this file for your constants?

Reply:
Done